chapters/inheritance.tex

```diff
@@ -1060,6 +1060,8 @@ \subsection{Restrictions on Redeclarations}\label{restrictions-on-redeclarations
 \subsection{Annotations for Redeclaration and Modification}\label{annotation-choices-for-suggested-redeclarations-and-modifications}
 
 A declaration can have an annotation \fmtannotationindex{choices} containing modifiers on \lstinline!choice!, where each of them indicates a suitable redeclaration or modifications of the element.
+The choices should use lookup from where they occur to find classes and constants, and should not reference protected classes that are globally inaccessible.
+It is possible to simplify the lookup by starting the names with a dot, see \cref{global-name-lookup}.
```

Comment:
The use of _should_ here feels ambiguous, but to me it doesn't sound like the resolution in the language group decision: https://github.com/modelica/ModelicaSpecification/issues/3244#issuecomment-1274750065

This would be more clear:
```suggestion
Lookup inside a \lstinline!choice! modifier is performed in the context of the annotation, meaning that references may need to be transformed to preserve the meaning when a \lstinline!choice! is applied in a different context.

\begin{nonnormative}
It is recommended to avoid expressions with references to elements that are not globally accessible, such as contents within a \lstinline!protected! section of a class.
By starting names with a dot it can be ensured that no transformation of references will be needed when a \lstinline!choice! is applied, and that applicability of a \lstinline!choice! does not depend on context, see \cref{global-name-lookup}.
\end{nonnormative}
```